ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -136,34 +128,82 @@ describe('HelpSidebar', () => {
   let runtimeStub: RuntimeApiStub;
   let appsStub: AppsApiStub;
   let props: {};
+  let user;
 
   const component = async () => {
-    const c = mountWithRouter(
-      <SWRConfig value={{ provider: () => new Map() }}>
+    /*
+    The useSWR hook in useGenomicExtractionJobs is causing
```

Comment:
oof.  good catch.

Reply:
Should we file a tech debt ticket to remove `swr` ?  looks like it's only used here.

Reply:
Thanks :-)

I made the tech debt ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-12935